src/NServiceBus.AcceptanceTests/Timeout/CyclingOutageTimeoutPersister.cs

```diff
@@ -95,7 +95,7 @@ public Task<TimeoutData> Peek(string timeoutId, ContextBag context)
             return Task.FromResult<TimeoutData>(null);
         }
 
-        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
+        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice, int maxChunkSize = Int32.MaxValue, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I'd say require the cancellation token as well. And then have the TM cancel it after lets say 10s by default?